From 932cd0c656bb8134a290316274f5d698e47d1a27 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <ikey@solus-project.com>
Date: Tue, 18 Apr 2017 01:49:18 +0100
Subject: [PATCH] wm: Default to an empty rpath

Signed-off-by: Ikey Doherty <ikey@solus-project.com>
---
 src/wm/meson.build | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/wm/meson.build b/src/wm/meson.build
index ae13e5e..c41a029 100644
--- a/src/wm/meson.build
+++ b/src/wm/meson.build
@@ -43,6 +43,8 @@ budgie_wm_vala_args =  [
         '--vapidir', dir_libconfig,
 ]
 
+rpath = ''
+
 if dep_mutter.version().version_compare('>=3.21.0')
     rpath = dep_mutter.get_pkgconfig_variable('typelibdir')
     budgie_wm_vala_args += ['--girdir', rpath]
-- 
2.7.4

